packages/web-app-files/src/mixins/actions/restore.ts

```diff
@@ -43,7 +51,126 @@ export default {
     ...mapMutations('runtime/spaces', ['UPDATE_SPACE_FIELD']),
     ...mapMutations(['SET_QUOTA']),
 
-    async $_restore_trigger({ resources }) {
+    getParentFolderFromResource(resource) {
+      return resource.path.slice(0, resource.path.lastIndexOf(resource.name))
+    },
+    getWebdavParentFolderFromResource(resource) {
+      const parentPath = this.getParentFolderFromResource(resource)
+      return buildWebDavFilesPath(this.user.id, parentPath)
```

Comment:
Erm... why the "old" webdav v2 in a hardcoded way? All our space objects from the runtime are capable of giving you a webDavPath (if needed at all), so that you don't need to know how to build a webDav path. This can't work for any space trashbin, only for personal spaces.